apps/mobile/src/screens/NftDetailScreen/NftAdditionalDetailsEth.tsx

```diff
@@ -59,16 +53,23 @@ export function NftAdditionalDetailsEth({ tokenRef }: NftAdditionalDetailsEthPro
     return null;
   }, [tokenMetadata]);
 
+  const numOfLinks = [openSeaExternalUrl, externalUrl, mirrorXyzUrl].filter((value) =>
+    value ? value : null
+  ).length;
```

Comment:
nit:
1. should `useMemo` whenever we operate on arrays to cache computation
2. for something like this you could probably use [Array.some()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some)

```js
const numLinks = useMemo(() => {
  return [openSeaExternalUrl, externalUrl, mirrorXyzUrl].some((el) => Boolean(el))
}, [openSeaExternalUrl, externalUrl, mirrorXyzUrl])
```